ods_ci/run_robot_test.sh

```diff
@@ -279,12 +279,12 @@ if command -v yq &> /dev/null
                 if [ -z "${SERVICE_ACCOUNT}" ]
                     then
                         echo "Performing oc login using username and password"
-                        oc_user=$(yq -er '.OCP_ADMIN_USER.USERNAME' "${TEST_VARIABLES_FILE}") || {
+                        oc_user=$(yq -e '.OCP_ADMIN_USER.USERNAME' "${TEST_VARIABLES_FILE}") || {
                             echo "Couldn't find '.OCP_ADMIN_USER.USERNAME' variable in provided '${TEST_VARIABLES_FILE}'."
                             echo "Please either provide it or use '--skip-oclogin true' (don't forget to login to the testing cluster manually then)."
                             exit 1
                         }
-                        oc_pass=$(yq -er '.OCP_ADMIN_USER.PASSWORD' "${TEST_VARIABLES_FILE}") || {
+                        oc_pass=$(yq -e '.OCP_ADMIN_USER.PASSWORD' "${TEST_VARIABLES_FILE}") || {
```

Comment:
This will change behavior for those who use the Python based `yq` implementaiton, see https://github.com/red-hat-data-services/ods-ci/issues/953:

```
$ yq -e '.OCP_ADMIN_USER.PASSWORD' "ods_ci/test-variables.yml"
"rhodsPW#123456"
$ yq -er '.OCP_ADMIN_USER.PASSWORD' "ods_ci/test-variables.yml"
rhodsPW#123456
$ yq --version                                                 
yq 3.2.3
```

Reply:
Also, the `-r` works for me even with the GoLang based implementation :thinking: :

```
$ /tmp/yq_linux_arm64 -er '.BROWSER.NAME' "ods_ci/test-variables.yml"
chrome
```